src/v/datalake/translation/partition_translator.cc

```diff
@@ -153,6 +157,8 @@ ss::future<> partition_translator::translate_until_stopped() {
       id);
 
     while (!_as.abort_requested()) {
+        co_await ss::sleep_abortable(_jitter.next_duration(), _as);
```

Comment:
shouldn't this be predicated on some condition? Like if there is an error in previous attempt, or am I missing something?

Reply:
Done